test/functional/tools/system/package_manager_test.py

```diff
@@ -25,8 +25,6 @@ def system_requirements(self):
                 print("missing:", not_installed)
         """)})
     client.run("create . --name=test --version=1.0 -s:b arch=armv8 -s:h arch=x86")
-    assert "dpkg-query: no packages found matching non-existing1:i386" in client.out
```

Comment:
I've removed these lines because `dpkg -l` doesn't print anything if it doesn't find any package with the `search pattern`